src/main/java/com/zharnasek/alesya/game/PeopleDemo.java

```diff
@@ -14,22 +14,45 @@ public static void main(String[] args) {
 
         while (true) {
             System.out.print("Input name, pose or activity('exit' to stop): ");
-            String num = scanner.next();
+            String num = scanner.next().trim();
 
-            if (num.trim().equals("exit")) {
+            if (num.equals("exit")) {
                 break;
             }
 
-            if (num.trim().equals("pose")){
+            if (num.equals("pose")){
                 Random rand = new Random();
                 int randomIndex = rand.nextInt(gameResources.getPoses().size());
                 gameResources.getPoses().get(randomIndex).descrOutput();
             }
 
-            if (num.trim().equals("activity")){
-                Random rand = new Random();
-                int randomIndex = rand.nextInt(gameResources.getActivities().size());
-                gameResources.getActivities().get(randomIndex).descrOutput();
+            if (num.equals("activity")){
+                System.out.print("Input name for him: ");
+                Gamer gamerMale = new Gamer(scanner.next(), Gamer.Gender.MALE);
+                System.out.print("Input name for her: ");
+                Gamer gamerFemale = new Gamer(scanner.next(), Gamer.Gender.FEMALE);
+                String activitiesMale;
+                String activitiesFemale;
+                int randomIndex;
+
+                do {
+                    Random rand = new Random();
```

Comment:
why do you need to create new object random in each cycle?

Reply:
updated